portfolio/src/main/webapp/index.html

```diff
@@ -1,27 +1,183 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Hadley Shapland</title>
-    <link rel="stylesheet" href="style.css">
-    <!-- include jquery -->
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="script.js"></script>
-  </head>
-
-  <body>
-    <div id="sidebar"></div>
- 
-    <div id="content">
-      <h1>HADLEY SHAPLAND</h1>
-      <p>Welcome to my webpage! Check out my <a href="https://www.linkedin.com/in/hadley-shapland/">
-          LinkedIn</a> and my <a href="https://github.com/hadleyshapland">
-          GitHub</a> to learn more about me.
-      </p>
-      <h3>Contact</h3>
-      <p>hadley.a.shapland@vanderbilt.edu</p>
-      <p>217.418.9903</p>
-      
-    </div>
-  </body>
+<html lang="en">
+   <head>
+      <meta charset="utf-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+      <meta name="description" content="Personal Webpage" />
+      <meta name="author" content="Hadley Shapland" />
+      <title>Hadley Shapland</title>
+      <link rel="apple-touch-icon" sizes="180x180" href="assets/img/apple-touch-icon.png">
+      <link rel="icon" type="image/png" sizes="32x32" href="assets/img/favicon-32x32.png">
+      <link rel="icon" type="image/png" sizes="16x16" href="assets/img/favicon-16x16.png">
+      <link rel="manifest" href="assets/img/site.webmanifest">
+      <!-- Font Awesome icons (free version)-->
+      <script src="https://use.fontawesome.com/releases/v5.13.0/js/all.js" crossorigin="anonymous"></script>
+      <!-- Google fonts-->
+      <link href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700" rel="stylesheet" type="text/css" />
+      <link href="https://fonts.googleapis.com/css?family=Muli:400,400i,800,800i" rel="stylesheet" type="text/css" />
+      <!-- Core theme CSS (includes Bootstrap)-->
+      <link href="css/styles.css" rel="stylesheet" />
```

Comment:
Consider using @import in your css/styles.css file to reduce the number of css links in the HTML. You can also import the bootstrap css directly from a CDN rather than copying it into a separate file.

e.g:
@import 'https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css'
@import 'https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700'

(see https://developer.mozilla.org/en-US/docs/Web/CSS/@import)

Reply:
I moved the fonts to the .css file using import. 

I tried to import the bootstrap css link directly, but because a lot of the colors and fonts are changed throughout the file from the template, it didn't work so I decided to keep the .css file as is.